### ISSUE TITLE
consider allowing volume to be configured for audio cues

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#202875]
In the meeting with @jooyoungseo, Venkatesh raised the point that some frequently played cues would be less abrasive if their volume was lower relatively speaking. He cited the error and inline suggestion cues as examples.

**Comment by hediet** [comment]
I think the audio cue files should have a balanced volume, there should be no need to adjust the volumes per audio cue.

**Comment by meganrogge** [comment]
@hediet, I think it makes sense that a user would want a very frequently played cue to be quieter than others.

**Comment by meganrogge** [comment]
If the audio cues are abrasive, we should fix that, not the volume. Thus closing.
